app/views/articles/index.html.erb

```diff
@@ -128,6 +128,11 @@
               <%= inline_svg("twitter-logo.svg", class:"icon-img") %>
             </a>
           <% end %>
+          <% if @user.mastodon_url.present? %>
```

Comment:
`@user` is actually a `User` or an `Organization` account in this view.

There's a check for class below.

(Would also accept refactorings to make this better 🙃)

Reply:
Silly me, had it down there at first and then moved it closer to the Twitter URL 🤦‍♂️ 

I don't have time right now for a refactor, but it's definitely something I'll consider doing in the future.

Reply:
Cool cool.

Reply:
The spec is fixed now, so this should be ready to merge. Thanks for the review @benhalpern!